### PR TITLE
Create net-core-prereqs-all-2.2.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
We probably don't need the 2.1 version #9751

### CODE REVIEW

File: aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2017 version 15.9 or later](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload
```
Comment (tdykstra) [inline aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md]: Should this be 15.9.3?

File: aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+* [Visual Studio Code](https://code.visualstudio.com/download)
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+* [C# for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
```
Comment (tdykstra) [inline aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md]: Not all tutorials include Mac sections -- we should have a ...all-no-mac or ...vs-vsc version.

Comment (Rick-Anderson) [reply aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md]: I agree. We should create _net-core-prereqs-vs-vsc-2.2.md_ with VS and VSC and then net-core-prereqs-all-2.2.md include would contain _net-core-prereqs-vs-vsc-2.2.md_ and the VS mac content.

Comment (tdykstra) [reply aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md]: Adding the heading sounds good to me.

File: aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md (hunk 1 above)
Comment (scottaddie) [reply aspnetcore/includes/aspnetcore/includes/net-core-prereqs-all-2.2.md]: I think 15.9 is correct.

**Pull request closed without merging**